CMakeLists.txt

```
@@ -15,11 +17,18 @@ OPTION(USE_TIMETAG "Set to ON to output time costs" OFF)
 OPTION(USE_DEBUG "Set to ON for Debug mode" OFF)
 OPTION(BUILD_STATIC_LIB "Build static library" OFF)
 OPTION(__BUILD_FOR_R "Set to ON if building lib_lightgbm for use with the R package" OFF)
+OPTION(__INTEGRATE_OPENCL "Set to ON if building lib_lightgbm with the OpenCL ICD Loader and its dependencies included" OFF)
```

According to the code, this option affects both targets `_lightgbm` (library) and `lightgbm` (executable).

```suggestion
OPTION(__INTEGRATE_OPENCL "Set to ON if building LightGBM with the OpenCL ICD Loader and its dependencies included" OFF)
```